test/jest/__mock__/reactFinalForm.mock.js

```diff
@@ -0,0 +1,18 @@
+jest.mock('react-final-form', () => ({
+  Field: ({ name, label, onChange, required, children, value }) => {
+    return (
+      <div>
+        <label htmlFor={name}>{label}</label>
+        <div
+          id={name}
+          name={name}
+          onChange={onChange}
+          required={required}
+        >
+          {children}
+          {value}
+        </div>
+      </div>
+    );
+  },
+}));
```

Comment:
We already have a file `finalFormComponents.mock.js` with that mock.

Reply:
@artem-blazhko - Changes updated